airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/ConnectionCell.tsx

```diff
@@ -12,43 +14,30 @@ interface ConnectionCellProps {
   destinationIcon?: string;
 }
 
-const Icon = styled(ConnectorIcon)`
-  margin-right: 12px;
-  display: inline-block;
-  vertical-align: middle;
-  height: 20px;
-  width: 20px;
-`;
-
-const Connector = styled.div`
-  font-weight: 500;
-  font-size: 14px;
-  line-height: 25px;
-`;
-
-const Arrow = styled(FontAwesomeIcon)`
-  margin: 0 17px 0 9px;
-  font-size: 9px;
-`;
-
 const ConnectionCell: React.FC<ConnectionCellProps> = ({
   sourceDefinitionName,
   destinationDefinitionName,
   sourceIcon,
   destinationIcon,
 }) => {
   return (
-    <>
-      <Connector>
-        <Icon icon={sourceIcon} />
-        {sourceDefinitionName}
-      </Connector>
-      <Connector>
-        <Arrow icon={faArrowRight} />
-        <Icon icon={destinationIcon} />
-        {destinationDefinitionName}
-      </Connector>
-    </>
+    <FlexContainer justifyContent="space-between">
```

Comment:
would love to use a grid setup for this and other bits related to table layout/sizing... but hesitate since none of our tables use it (yet)